### PR TITLE
update: sandboxing and new lib

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

**Comment by abdulowork** [comment]
LGTM

File: docs/topics/native/native-cocoapods-xcode.md [hunk 1]
```diff
@@ -22,10 +22,15 @@ dependency by calling `pod install` manually for each Xcode project. In other ca
 ## Xcode project with one target
 
 1. Create an Xcode project with a `Podfile` if you haven't done so yet.
-2. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle(.kts)`
+2. Make sure to disable the **User Script Sandboxing** under **Build Options** in the application target:
```
Comment (sarahhaggarty) [inline docs/topics/native/native-cocoapods-xcode.md:25]:
```suggestion
2. Make sure to disable **User Script Sandboxing** under **Build Options** in the application target:
```

File: docs/topics/native/native-cocoapods-xcode.md [hunk 2]
```diff
@@ -22,10 +22,15 @@ dependency by calling `pod install` manually for each Xcode project. In other ca
...
+
+   ![Disable sandboxing CocoaPods](disable-sandboxing-cocoapods.png)
+
+3. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle(.kts)`
```
Comment (sarahhaggarty) [inline docs/topics/native/native-cocoapods-xcode.md:29]:
```suggestion
3. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` in the `build.gradle(.kts)` file
```

**Review by sarahhaggarty — APPROVED** [review]
Looks great! Just two very minor suggestions

**Pull request merged**